qr-backup

```diff
@@ -1276,28 +1277,28 @@ def main_backup(args):
     )
 
     # PIL's pdf writer needs to mmap, so it can't accept sys.stdout directly
-    tmp_pdf = io.BytesIO()
-    pages[0].save(tmp_pdf, format="pdf", save_all=True, append_images=pages[1:], resolution=dpi, producer="qr-backup", title=f"qr-backup paper backup of {restore_file} with sha256 {sha256sum} and length {original_len}", creationDate=backup_date, modDate=backup_date)
+    tmp_file = io.BytesIO()
+    pages[0].save(tmp_file, format="tiff", compression="tiff_lzw", save_all=True, append_images=pages[1:], resolution=dpi, producer="qr-backup", title=f"qr-backup paper backup of {restore_file} with sha256 {sha256sum} and length {original_len}", creationDate=backup_date, modDate=backup_date)
```

Comment:
Confirmed that the LZW patent has expired, so this should be okay for open-source